script/ghIssues.pl

```diff
@@ -96,6 +103,7 @@ sub getIssues{
                 last_update => $issue->{updated_at},
                 last_commit => $last_commit,
                 last_comment => $last_comment,
+                producer => $issue->{assignee}->{login} || undef,
```

Comment:
Well that would only matter for the IA pages.  If we really want I could randomly assign someone and then push that update to the PR on github too.